### PR TITLE
update C++ filters

### PR DESCRIPTION
**Pull request by hkctkuy**

### CODE REVIEW

File: libcasr/src/constants.rs [hunk 1]
```diff
@@ -194,6 +194,17 @@ pub const STACK_FRAME_FUNCTION_IGNORE_REGEXES_CPP: &[&str] = &[
...
     r".*stdext::exception::what",
     r".*v8::base::OS::Abort",
+    // Pybindings
+    r".*pybind.*",
+    r"^PyCFunction.*",
+    r"^PyObject.*",
+    r"^PyEval.*",
+    r"^PyRun.*",
+    r"^Py_.*",
+    r"^_start",
```
Comment (SweetVishnya) [inline libcasr/src/constants.rs:204]: This is already defined

File: libcasr/src/constants.rs [hunk 2]
```diff
@@ -194,6 +194,17 @@ pub const STACK_FRAME_FUNCTION_IGNORE_REGEXES_CPP: &[&str] = &[
...
+    r"^__libc_start_main",
```
Comment (SweetVishnya) [inline libcasr/src/constants.rs:205]: can you move it close to `_start`?

File: libcasr/src/constants.rs [hunk 3]
```diff
@@ -194,6 +194,17 @@ pub const STACK_FRAME_FUNCTION_IGNORE_REGEXES_CPP: &[&str] = &[
...
+    r"^fuzzer::.*",
```
Comment (SweetVishnya) [inline libcasr/src/constants.rs:206]: already defined

File: libcasr/src/constants.rs [hunk 4]
```diff
@@ -194,6 +194,17 @@ pub const STACK_FRAME_FUNCTION_IGNORE_REGEXES_CPP: &[&str] = &[
     r".*logging::LogMessage",
     r".*stdext::exception::what",
     r".*v8::base::OS::Abort",
+    // Pybindings
+    r".*pybind.*",
+    r"^PyCFunction.*",
+    r"^PyObject.*",
+    r"^PyEval.*",
+    r"^PyRun.*",
+    r"^Py_.*",
```
Comment (SweetVishnya) [inline libcasr/src/constants.rs:203]: .* is redundant on right side (and in other places)

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

**Comment by SweetVishnya** [comment]
We should also verify this PR influence on pytorch-py OSS-Sydr-Fuzz project.

File: libcasr/src/constants.rs (hunk 1 above)
Comment (hkctkuy) [reply libcasr/src/constants.rs]: Done

File: libcasr/src/constants.rs (hunk 2 above)
Comment (hkctkuy) [reply libcasr/src/constants.rs]: Done

File: libcasr/src/constants.rs (hunk 3 above)
Comment (hkctkuy) [reply libcasr/src/constants.rs]: Done

File: libcasr/src/constants.rs (hunk 4 above)
Comment (hkctkuy) [reply libcasr/src/constants.rs]: Done

**Pull request merged**